aten/src/ATen/Context.cpp

```diff
@@ -66,5 +66,10 @@ bool Context::hasCUDA() const {
 #endif
 }
 
+#ifdef AT_CUDA_ENABLED
+cudaStream_t Context::getCurrentCUDAStream() const {
```

Comment:
Yay! :)

Reply:
@killeent can you look into this?

Reply:
Bleh, this one might be tricky because the method is on Context. I'd be careful about perf problems that result from trying to init cuda in this method, because it requires taking a lock to initialize CUDA. Other pathways avoid it because, for instance, you cannot get a CUDA backend type without initializing cuda and so all tensor/type operators are guaranteed that cuda is initialized.

Hence, for any native function that is working on CUDA tensors, CUDA is guaranteed initialized and this is safe to call, but for public uses of the API this may fail.

Reply:
OK. This is worth commenting somewhere inside `ATen/Context.h`

Reply:
Yeah I thought about initialization but agreed with @zdevito that it would be messy, and I'm assuming that if you have a CUDA tensor in your function, you've already initialized CUDA in order to create said Tensor.